local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -45,7 +46,7 @@ type VolumeUtil interface {
 	GetFsCapacityByte(fullPath string) (int64, error)
 
 	// Get capacity of the block device
-	GetBlockSpaceByte(fullPath string) (uint64, error)
+	GetBlockSpaceByte(fullPath string) (int64, error)
```

Comment:
I'd recommend GetBlockCapacityByte for this method. This name is consistent with GetFsCapacityByte.